src/settings/time.tsx

```diff
@@ -56,3 +55,9 @@ export class Time implements TimeDate {
 }
 
 export const time = new Time([0], [1], [5]);
+
+export const hours = time.hours;
```

Comment:
could also be `export const {hours} = time`

Reply:
changed